src/main.c

```diff
@@ -36,16 +37,30 @@ record_env(FILE *fout, char **ep)
 	(void) fprintf(fout, "%s\n", *p);
 }
 
+char **
+handle_options(char **argv)
+{
+}
+
 int
 main(int argc, char **argv, char **envp)
 {
     if (argc < 2)
 	error(EX_USAGE, 0, "missing command to record");
 
-    record_start("build-recorder.out");
+    char *output_fname = "build-recorder.out";
+
+    if (!strcmp(argv[1], "-o")) {
+	output_file = argv[2];
```

Comment:
did you even compile it?
I'm pretty sure it has to be `output_fname` here

Reply:
I wanted to but main doesn't compile anyways because of an error in another file, i am opening a pull request now to fix it. How and when did it pass through? I will add a reminder command on the pre-commit hook. Can't afford to keep forgetting to compile and test, even for minor changes.